### ISSUE TITLE
Error parsing SQLite statement containing operators.

### ISSUE THREAD

**Issue by relic-yuexi** [macbre/sql-metadata#519]
```
from sql_metadata import Parser

parser = Parser("select 老年人口占比 * 人口数量 from 城市 where 人口数量 = (select max(人口数量) from 城市)")

print(parser.columns)

print(parser.tables)
```

![image](https://github.com/user-attachments/assets/4bb25038-3c6d-49b3-bdbe-9d3c9508f912)

![image](https://github.com/user-attachments/assets/9486fba2-4102-4843-9955-f7b51fd24a5c)

![image](https://github.com/user-attachments/assets/db77a2b9-df3a-4b78-b7ea-de4c28264731)

![image](https://github.com/user-attachments/assets/f5f7a610-556e-4768-8701-f3d5764343cd)

![image](https://github.com/user-attachments/assets/c8f29cbc-298e-492d-9ef5-11529472f3fb)

**Comment by MiuNice** [comment]
I submitted a PR on this issue, which can be used before the official release.
PR: #523 .

**Comment by macbre** [comment]
Resolved by #523.
